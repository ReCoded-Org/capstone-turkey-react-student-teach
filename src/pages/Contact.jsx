import { useFormik } from 'formik';
import {
  FaInstagram,
  FaGithub,
  FaLinkedinIn,
  FaYoutube,
  FaMapMarker,
  FaPhoneAlt,
  FaMediumM,
} from 'react-icons/fa';

function Contact() {
  const formik = useFormik({
    initialValues: {
      name: '',
      email: '',
      text: '',
      checked: [],
    },
    onSubmit: (values) => {
      console.log(values);
    },
  });

  return (
    <div>
      <section className="mt-5">
        <form
          onSubmit={formik.handleSubmit}
          className="flex flex-col justify-center items-center bg-red-200"
        >
          <label htmlFor="name" className="flex flex-col items-start mt-7">
            Name
            <input
              id="name"
              type="text"
              placeholder="Your name"
              onChange={formik.handleChange}
              value={formik.values.name}
              className="w-[11.3rem] p-1  mt-2 text-[12px] text-left"
            />
          </label>
          <label htmlFor="email" className="flex flex-col items-start mt-7">
            Email
            <input
              id="email"
              type="email"
              placeholder="Your email"
              onChange={formik.handleChange}
              value={formik.values.email}
              className="w-[11.3rem] p-1  mt-2 text-[12px] text-left"
            />
          </label>
          <label htmlFor="text" className="flex flex-col items-start mt-7">
            How can we help?
            <textarea
              id="text"
              type="text"
              placeholder="Tell us about the project..."
              onChange={formik.handleChange}
              value={formik.values.text}
              className="w-[11.3rem] p-1 pb-[8rem] mt-2 text-[12px] text-left"
            />
          </label>
          <div>
            <label htmlFor="website-design">
              Website Design
              <input
                type="checkbox"
                name="checked"
                value="website-design"
                id="website-design"
              />
            </label>
          </div>

          <button type="submit" className="px-[4.13rem] py-1 mt-2 bg-cusOrange">
            Submit
          </button>
        </form>
      </section>

      <section className="mt-5 bg-cusOrange">
        <div className="p-4">
          <h1 className="font-bold">Get in touch</h1>
          <h4 className="mt-2 ">
            We love to hear from you, Our team is always here to chat.
          </h4>
        </div>
        <div className="flex flex-col bg-cusOrange p-4 pt-3">
          <div className="flex justify-center items-center mr-3">
            <FaMediumM className="" />
            <h2 className="font-bold ml-2">Email</h2>
          </div>
          <h3>Contact us via email</h3>
          <h3>hi@untitiledui.com</h3>
        </div>

        <div className="flex flex-col bg-cusOrange p-4">
          <div className="flex justify-center items-center mr-3">
            <FaPhoneAlt className=" " />
            <h2 className="font-bold ml-2">Phone</h2>
          </div>
          <h3>Contact us via phone</h3>
          <h3>552-555-5555</h3>
        </div>
        <div className="flex flex-col bg-cusOrange p-4">
          <div className="flex justify-center items-cente ">
            <FaMapMarker className="" />
            <h2 className="font-bold ml-2">Location</h2>
          </div>
          <h3>Turkey, Ankara</h3>
          <h3>36 Street, Bahcelievler</h3>
        </div>

        <div className="flex justify-center items-center p-3">
          <FaInstagram className="mx-3 text-xl" />
          <FaGithub className="mx-3 text-xl" />
          <FaLinkedinIn className="mx-3 text-xl" />
          <FaYoutube className="mx-3 text-xl" />
        </div>
      </section>
    </div>
  );
}

export default Contact;
