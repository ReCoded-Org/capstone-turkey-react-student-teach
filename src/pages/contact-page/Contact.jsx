import { useFormik } from 'formik';

function Contact() {
  const formik = useFormik({
    initialValues: {
      name: '',
      email: '',
      text: '',
      checked: [],
    },
    onSubmit: (values) => {
      console.log(values);
    },
  });

  return (
    <section className="mt-5">
      <form
        onSubmit={formik.handleSubmit}
        className="flex flex-col justify-center items-center bg-red-200"
      >
        <label htmlFor="name" className="flex flex-col items-start mt-7">
          Name
          <input
            id="name"
            type="text"
            placeholder="Your name"
            onChange={formik.handleChange}
            value={formik.values.name}
            className="w-[11.3rem] p-1  mt-2 text-[12px] text-left"
          />
        </label>
        <label htmlFor="email" className="flex flex-col items-start mt-7">
          Email
          <input
            id="email"
            type="email"
            placeholder="Your email"
            onChange={formik.handleChange}
            value={formik.values.email}
            className="w-[11.3rem] p-1  mt-2 text-[12px] text-left"
          />
        </label>
        <label htmlFor="text" className="flex flex-col items-start mt-7">
          How can we help?
          <textarea
            id="text"
            type="text"
            placeholder="Tell us about the project..."
            onChange={formik.handleChange}
            value={formik.values.text}
            className="w-[11.3rem] p-1 pb-[8rem] mt-2 text-[12px] text-left"
          />
        </label>
        <div>
          <label htmlFor="website-design">
            Website Design
            <input
              type="checkbox"
              name="checked"
              value="website-design"
              id="website-design"
            />
          </label>
        </div>

        <button type="submit" className="px-[4.13rem] py-1 mt-2 bg-cusOrange">
          Submit
        </button>
      </form>
    </section>
  );
}

export default Contact;
