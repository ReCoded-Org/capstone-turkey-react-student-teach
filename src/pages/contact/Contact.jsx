import {
  FaInstagram,
  FaGithub,
  FaLinkedinIn,
  FaYoutube,
  FaMapMarker,
  FaPhoneAlt,
  FaMediumM,
  FaSpinner,
} from 'react-icons/fa';
import { Formik, Form } from 'formik';
import * as Yup from 'yup';
import FormField from '../../components/FormField/FormField';

const SignInSchema = Yup.object().shape({
  username: Yup.string().min(3).max(24).required(),
  email: Yup.string().email().required(),
  password: Yup.string().min(8).max(32).required(),
  passwordConfimation: Yup.string()
    .oneOf([Yup.ref('password'), null], 'Passwords does not match')
    .required(),
});
function Contact() {
  return (
    <div className="lg:flex lg:flex-row-reverse lg:justify-end lg:h-screen">
      <section>
        <Formik
          initialValues={{
            name: '',
            password: '',
          }}
          validationSchema={SignInSchema}
          onSubmit={(values, { setSubmitting }) => {
            setTimeout(() => {
              console.log(values, 'aa');
              setSubmitting(false);
            }, 100);
          }}
        >
          {({
            // Unused props: handleSubmit, handleChange, handleBlur, values, isValid,
            touched,
            errors,
            isSubmitting,
            // handleSubmit,
          }) => (
            <Form>
              <FormField
                autoComplete="username"
                className="mb-3 "
                errors={errors}
                label="Name"
                name="name"
                touched={touched}
                type="text"
              />

              {isSubmitting ? (
                <button
                  type="submit"
                  className="text-lg bg-red-700 text-gray-200 rounded flex items-center px-5 py-2"
                  disabled
                >
                  <FaSpinner className="animate-spin -ml-1 mr-3 h-5 w-5 text-white" />
                  Submiting...
                </button>
              ) : (
                <button
                  type="submit"
                  className="text-lg text-gray-100 bg-red-600 hover:bg-red-700 transition rounded px-5 py-2"
                >
                  Submit
                </button>
              )}
            </Form>
          )}
        </Formik>
      </section>

      <section className="bg-cusOrange lg:flex lg:flex-col lg:items-start lg:pr-[2rem] ">
        <div className="p-4 lg:mt-[5rem] lg:text-left lg:ml-10">
          <h1 className="font-bold lg:text-xl">Get in touch</h1>
          <h4 className="mt-2 lg:mt-1">
            We love to hear from you,
            <br /> Our team is always here to chat.
          </h4>
        </div>
        <div className="flex flex-col bg-cusOrange p-4 lg:flex lg:flex-row lg:items-start lg:text-left lg:ml-10 lg:mt-5">
          <div className="flex justify-center items-center mr-3">
            <FaMediumM className="lg:mt-1" />
            <h2 className="font-bold ml-2 block lg:hidden">Email</h2>
          </div>
          <div className="lg:flex lg:flex-col lg:items-start">
            <h2 className="font-bold ml-2 lg:m-0 lg:block hidden">Email</h2>
            <h3 className="lg:mt-1">Contact us via email</h3>
            <h3>hi@untitiledui.com</h3>
          </div>
        </div>

        <div className="flex flex-col bg-cusOrange p-4 lg:flex lg:flex-row lg:items-start lg:text-left lg:ml-10 lg:mt-5">
          <div className="flex justify-center items-center mr-3">
            <FaPhoneAlt className="lg:mt-1" />
            <h2 className="font-bold ml-2 block lg:hidden">Phone</h2>
          </div>
          <div className="lg:flex lg:flex-col lg:items-start">
            <h2 className="font-bold ml-2 lg:m-0 lg:block hidden">Phone</h2>
            <h3 className="lg:mt-1">Contact us via phone</h3>
            <h3>552-555-5555</h3>
          </div>
        </div>
        <div className="flex flex-col bg-cusOrange p-4 lg:flex lg:flex-row lg:items-start lg:text-left lg:ml-10 lg:mt-5">
          <div className="flex justify-center items-center mr-3">
            <FaMapMarker className="lg:mt-1" />
            <h2 className="font-bold ml-2 lg:m-0 block lg:hidden">Location</h2>
          </div>
          <div className="lg:flex lg:flex-col lg:items-start">
            <h2 className="font-bold ml-2 lg:m-0 lg:block hidden">Location</h2>
            <h3 className="lg:mt-1">Turkey, Ankara</h3>
            <h3>36 Street, Bahcelievler</h3>
          </div>
        </div>
        <div className="flex justify-center items-center p-3 lg:mt-[19rem] lg:ml-[1rem]">
          <FaInstagram className="social-icons" />
          <FaGithub className="social-icons" />
          <FaLinkedinIn className="social-icons" />
          <FaYoutube className="social-icons" />
        </div>
      </section>
    </div>
  );
}

export default Contact;
