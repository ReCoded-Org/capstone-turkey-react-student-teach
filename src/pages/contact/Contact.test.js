import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Contact from './Contact';

it('Navbar snapshot', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Contact />
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
