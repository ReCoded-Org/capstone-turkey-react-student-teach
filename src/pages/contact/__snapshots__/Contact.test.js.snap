// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar snapshot 1`] = `
<div
  className="lg:flex lg:flex-row-reverse lg:justify-end lg:h-screen lg:text-base md:text-base text-sm"
>
  <section
    className="w-screen h-3/4 mt-10 md:h-[60vh] flex flex-col justify-center items-center lg:mt-[10rem]"
  >
    <h1
      className="mb-10 font-semibold text-2xl md:text-3xl lg:text-[2.3rem] lg:mb-[3.5]"
    >
      Level up Your brand
    </h1>
    <form
      action="#"
      className="w-[15rem] lg:w-[20rem]"
      onReset={[Function]}
      onSubmit={[Function]}
    >
      <div
        className="mb-10 lg:text-left"
      >
        <label
          className="block text-sm font-medium text-gray-700"
          htmlFor="name"
        >
          Name
        </label>
        <input
          autoComplete="name"
          className="my-1 focus:ring-cusOrange focus:border-cusOrange block w-full shadow-sm sm:text-sm border-gray-300 rounded-md"
          id="name"
          name="name"
          onBlur={[Function]}
          onChange={[Function]}
          placeholder="Your Name"
          type="text"
          value=""
        />
      </div>
      <div
        className="mb-10 lg:text-left"
      >
        <label
          className="block text-sm font-medium text-gray-700"
          htmlFor="email"
        >
          Email
        </label>
        <input
          autoComplete="email"
          className="my-1 focus:ring-cusOrange focus:border-cusOrange block w-full shadow-sm sm:text-sm border-gray-300 rounded-md"
          id="email"
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          placeholder="You@company.com"
          type="email"
          value=""
        />
      </div>
      <div
        className="mb-3 lg:text-left"
      >
        <label
          className="block text-sm font-medium text-gray-700"
          htmlFor="message"
        >
          How can we help?
        </label>
        <textarea
          autoComplete="name"
          className="my-1 focus:ring-cusOrange focus:border-cusOrange block w-full shadow-sm sm:text-sm border-gray-300 rounded-md lg:pb-[7rem]"
          id="message"
          name="message"
          onBlur={[Function]}
          onChange={[Function]}
          placeholder="Tell us about your project"
          rows="5"
        />
      </div>
      <h1
        className="text-left font-medium"
      >
        Services
      </h1>
      <div
        className="flex my-5 w-max lg:ml-5"
      >
        <div
          className="flex flex-col items-start mr-5 w-max"
        >
          <div
            className="contact-checkbox-field"
          >
            <label
              className="text-sm font-medium text-gray-700"
              htmlFor="service"
            >
              Web Design
            </label>
            <input
              checked={false}
              className="mr-2 text-cusOrange focus:ring-0"
              id="service"
              name="service"
              onBlur={[Function]}
              onChange={[Function]}
              type="checkbox"
              value="WebDesign"
            />
          </div>
          <div
            className="contact-checkbox-field"
          >
            <label
              className="text-sm font-medium text-gray-700"
              htmlFor="service"
            >
              Content Creation
            </label>
            <input
              checked={false}
              className="mr-2 text-cusOrange focus:ring-0"
              id="service"
              name="service"
              onBlur={[Function]}
              onChange={[Function]}
              type="checkbox"
              value="ContentCreation"
            />
          </div>
        </div>
        <div
          className="flex flex-col items-start w-max"
        >
          <div
            className="contact-checkbox-field"
          >
            <label
              className="text-sm font-medium text-gray-700"
              htmlFor="service"
            >
              UX Desgin
            </label>
            <input
              checked={false}
              className="mr-2 text-cusOrange focus:ring-0"
              id="service"
              name="service"
              onBlur={[Function]}
              onChange={[Function]}
              type="checkbox"
              value="UxDesgin"
            />
          </div>
          <div
            className="contact-checkbox-field"
          >
            <label
              className="text-sm font-medium text-gray-700"
              htmlFor="service"
            >
              Other
            </label>
            <input
              checked={false}
              className="mr-2 text-cusOrange focus:ring-0"
              id="service"
              name="service"
              onBlur={[Function]}
              onChange={[Function]}
              type="checkbox"
              value="Other"
            />
          </div>
        </div>
      </div>
      <button
        className="text-lg text-gray-100 bg-cusOrange transition rounded px-8 py-2 mb-10 lg:mb-0 hover:scale-110 ease-in-out"
        type="submit"
      >
        Submit
      </button>
    </form>
  </section>
  <section
    className="bg-cusOrange mt-10 md:h-3/4 lg:h-[100vh] lg:mt-0 lg:flex lg:flex-col lg:items-start lg:pr-[2rem]"
  >
    <div
      className="p-4 lg:mt-[5rem] lg:text-left lg:ml-10"
    >
      <h1
        className="font-bold lg:text-xl"
      >
        Get in touch
      </h1>
      <h4
        className="mt-2 lg:mt-1"
      >
        We love to hear from you,
        <br />
         Our team is always here to chat.
      </h4>
    </div>
    <div
      className="contact-info-icon-container"
    >
      <div
        className="contact-info-icon"
      >
        <svg
          className="lg:mt-1"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 512 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M71.5 142.3c.6-5.9-1.7-11.8-6.1-15.8L20.3 72.1V64h140.2l108.4 237.7L364.2 64h133.7v8.1l-38.6 37c-3.3 2.5-5 6.7-4.3 10.8v272c-.7 4.1 1 8.3 4.3 10.8l37.7 37v8.1H307.3v-8.1l39.1-37.9c3.8-3.8 3.8-5 3.8-10.8V171.2L241.5 447.1h-14.7L100.4 171.2v184.9c-1.1 7.8 1.5 15.6 7 21.2l50.8 61.6v8.1h-144v-8L65 377.3c5.4-5.6 7.9-13.5 6.5-21.2V142.3z"
          />
        </svg>
        <h2
          className="font-bold ml-2 block lg:hidden"
        >
          Email
        </h2>
      </div>
      <div
        className="lg:flex lg:flex-col lg:items-start"
      >
        <h2
          className="font-bold ml-2 lg:m-0 lg:block hidden"
        >
          Email
        </h2>
        <h3
          className="lg:mt-1"
        >
          Contact us via email
        </h3>
        <h3>
          hi@untitiledui.com
        </h3>
      </div>
    </div>
    <div
      className="contact-info-icon-container"
    >
      <div
        className="contact-info-icon"
      >
        <svg
          className="lg:mt-1"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 512 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M497.39 361.8l-112-48a24 24 0 0 0-28 6.9l-49.6 60.6A370.66 370.66 0 0 1 130.6 204.11l60.6-49.6a23.94 23.94 0 0 0 6.9-28l-48-112A24.16 24.16 0 0 0 122.6.61l-104 24A24 24 0 0 0 0 48c0 256.5 207.9 464 464 464a24 24 0 0 0 23.4-18.6l24-104a24.29 24.29 0 0 0-14.01-27.6z"
          />
        </svg>
        <h2
          className="font-bold ml-2 block lg:hidden"
        >
          Phone
        </h2>
      </div>
      <div
        className="lg:flex lg:flex-col lg:items-start"
      >
        <h2
          className="font-bold ml-2 lg:m-0 lg:block hidden"
        >
          Phone
        </h2>
        <h3
          className="lg:mt-1"
        >
          Contact us via phone
        </h3>
        <h3>
          552-555-5555
        </h3>
      </div>
    </div>
    <div
      className="contact-info-icon-container"
    >
      <div
        className="contact-info-icon"
      >
        <svg
          className="lg:mt-1"
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": undefined,
            }
          }
          viewBox="0 0 384 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M172.268 501.67C26.97 291.031 0 269.413 0 192 0 85.961 85.961 0 192 0s192 85.961 192 192c0 77.413-26.97 99.031-172.268 309.67-9.535 13.774-29.93 13.773-39.464 0z"
          />
        </svg>
        <h2
          className="font-bold ml-2 lg:m-0 block lg:hidden"
        >
          Location
        </h2>
      </div>
      <div
        className="lg:flex lg:flex-col lg:items-start"
      >
        <h2
          className="font-bold ml-2 lg:m-0 lg:block hidden"
        >
          Location
        </h2>
        <h3
          className="lg:mt-1"
        >
          Turkey, Ankara
        </h3>
        <h3>
          36 Street, Bahcelievler
        </h3>
      </div>
    </div>
    <div
      className="flex justify-center items-center p-3 lg:mt-[19rem] lg:ml-[1rem]"
    >
      <svg
        className="contact-social-icons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
        />
      </svg>
      <svg
        className="contact-social-icons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 496 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
        />
      </svg>
      <svg
        className="contact-social-icons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
        />
      </svg>
      <svg
        className="contact-social-icons"
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 576 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"
        />
      </svg>
    </div>
  </section>
</div>
`;
